tests/test_model_methods.py

```diff
@@ -255,6 +255,33 @@ def test_setup_reservoirs(source, tmpdir, example_wflow_model):
 number of reservoirs in model area"
 
 
+def test_setup_ksathorfrac(tmpdir, example_wflow_model):
+    # Read the modeldata
+    model = "wflow"
+    example_wflow_model.read()
+    # Add the wflow datacatalog
+    example_wflow_model.data_catalog.from_yml(
+        "https://raw.githubusercontent.com/Deltares/hydromt_wflow/setup_ksat/data/catalogs/wflow_data.yml"
```

Comment:
See the general comment on adding a shortcut for this catalog. For testing, there is no p drive access so I suggest to export the ksathorfrac maps for the Piave extent for testing.
And add one additional test to check if the wflow_data catalog can be read correctly (as we cannot go further and access the data).